typings/contentfulManagement.d.ts

```diff
@@ -0,0 +1,40 @@
+import { Agent as httpAgent } from "http"
+import { Agent as httpsAgent } from "https"
+import { AxiosProxyConfig } from "axios"
+import { Stream } from "stream"
+import { User } from "./user"
+import { DefaultElements } from "./defaultElements"
+import { PersonalAccessToken, PersonalAccessTokenProp } from "./personalAccessToken"
+import { Space, SpaceProps } from "./space"
+import { Collection } from "./collection"
+import { Organisation } from "./organisation"
+
+export as namespace contentfulManagementStatic
+
+declare function createClient(params: ClientParams): ClientAPI
+
+export interface ClientParams {
+  accessToken: string,
+  insecure?: boolean | null,
+  retryOnError?: boolean | null,
+  host?: null | string,
+  hostUpload?: null | string,
+  httpAgent?: null | httpAgent,
+  httpsAgent?: null | httpsAgent,
+  proxy?: null | AxiosProxyConfig,
+  headers?: null | { [key: string]: any },
+  logHandler?: (level: string, data: Error | string) => void,
+  application?: null | string,
+  integration?: null | string,
+  timeout?: number,
+}
+
+export interface ClientAPI {
```

Comment:
I think we are missing some methods here

```
getSpaces
getUsagePeriods
getUsages
rawRequest
```

Reply:
You right. I will add it somewhere today when I have some spare minutes.

Reply:
@Khaledgarbaya is it correct that getUsagePeriods throws always a 404?